client/scss/layouts/_compare-revisions.scss

```diff
@@ -1,9 +1,21 @@
-$color-addition-dark: #a6f3a6;
-$color-addition-light: #ebffeb;
-$color-deletion-dark: #f8cbcb;
-$color-deletion-light: #ffebeb;
+$color-addition-dark: var(--color-addition-dark);
+$color-addition-light: var(--color-addition-light);
+$color-deletion-dark: var(--color-deletion-dark);
+$color-deletion-light: var(--color-deletion-light);
 
 .comparison {
+  --color-addition-dark: #a6f3a6;
+  --color-addition-light: #ebffeb;
+  --color-deletion-dark: #f8cbcb;
```

Comment:
Is there any chance of making thins brighter. I found it quite muted and easy to miss in the screenshot 👓

Reply:
For now I’d prefer to keep this as-is. None of the colors are technically suitable for their current usage, they all need changing (as well as having a signifier that’s not based on color). I’d prefer not to hold up the patch release with this.

Reply:
👍🏼 and ❤️